Feature/task 14 pozdeeva

OxanaPozdeeva opened the pull request.

dha01 committed:
Admin: Уманов А.А. : тестовый запрос

egorovk committed:
Егоров Кирилл Тестовый запрос

DenNikonov committed:
Никонов тест

egorovk committed:
test

Signed-off-by: EgorovKirill <d4bit@yandex.ru>

OxanaPozdeeva committed:
testing

DenNikonov committed:
Nikonov Test Push

nikolayozhogov committed:
test commit from nikolay ozhogov

OxanaPozdeeva committed:
new_testing

OxanaPozdeeva committed:
newnew_testing

OxanaPozdeeva committed:
testtesttest

Tasik73 committed:
Test_Turisina_task_0

egorovk committed:
test commit from Egorov Kirill

Signed-off-by: EgorovKirill <d4bit@yandex.ru>

dha01 committed:
Merge pull request #5 from dha01/feature/Task0_Test_admin

Admin: Уманов А.А. : тестовый запрос

dha01 committed:
Merge pull request #11 from dha01/feature/task_0_create_project_TT

Test_Turisina_task_0

dha01 committed:
Merge pull request #10 from dha01/feature/OxanaPozdeeva

Feature/oxana pozdeeva

dha01 committed:
Merge pull request #4 from dha01/nikolayozhogov

test commit from nikolay ozhogov

dha01 committed:
Merge pull request #8 from dha01/feature/Nikonov_test

Feature/nikonov test

dha01 committed:
Merge pull request #12 from dha01/2203

2203

OxanaPozdeeva committed:
Merge branch 'master' of https://github.com/dha01/IS

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team

Написала запрос на создание схемы Team.team
Написала запрос на создание таблицы Team
Написала запрос на добавление записей в таблицу Team
Добавила 2 записи с нашими группами

OxanaPozdeeva committed:
test

OxanaPozdeeva committed:
delete

dha01 commented on EgorovKtest/EgorovKtest.txt:
```diff
@@ -0,0 +1 @@
+test
```
По всей видимости, перед созданием ветки (и в принципе давно) не был сделан Pull в master и так же закоммичены все файлы подряд.
Нужно привести ветку(и локальный репозиторий) в порядок, чтобы остались только относящиеся к задаче файл/файлы

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+		'��-22�',
+		'10.10.2011'
+		)
+	 
```
Папка должна называться Task_14
Название файла должно содержать краткое описание содержимого на английском(create table Team.team). Фамилия в названии файла не нужна.

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+		data)
+	values
+		(
+		'��-22�',
```
Кодировка файла должна быть "Unicode ... 65001"

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+	team int identity,
+	name varchar(50) not null,
+	data datetime,
+	programma int,
```
Ссылка на таблицу Specialty.specialty_detail нужна заменить все названия на соответствующие.

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+	data datetime,
+	programma int,
+	constraint PK__team primary key (team),
+	constraint FK__team__programma foreign key (programma) references Team.team
```
references <Tablename> указание на таблицу на которую указывает вторичный ключ т.е. на Specialty.specialty_detail
Остальные поля так же заменить на соответствующие.

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+create schema Team
+
+create table Team.team
```
-- Комментарий к таблице

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+create schema Team
```
-- Комментарий к схеме

dha01 commented on SQL/Task__14/TeamPozdeeva.sql:
```diff
@@ -0,0 +1,21 @@
+(
+	team int identity,
+	name varchar(50) not null,
+	data datetime,
```
Не нужно путать:
data - данные
date - дата
И лучше будет назвать create_date и сделать с типом date, тут нас время не интересует.

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Произведены изменения в соответствии с комментариями.
Изменение кодировки.
Изменение названия файла.
Изменение типа поля дата.
Изменение поля и ссылки на таблицу Specialty.specialty_detail.

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+	data date,
+	specialty int,
+	constraint PK__team primary key (team),
+	constraint FK__team__specialty foreign key (specialty) references Specialty.specialty_detail
```
FK__team__specialty_detail
foreign key (specialty_detail)

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+﻿create schema Team
```
Добавить комментарий.

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+﻿create schema Team
+
+create table Team.team
```
Добавить комментарий.

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+	team int identity,
+	name varchar(50) not null,
+	data date,
+	specialty int,
```
specialty_detail

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+(
+	team int identity,
+	name varchar(50) not null,
+	data date,
```
Почему поле с датой создания называется "данные"(data)?
create_date

dha01 commented on .gitattributes:
```diff
@@ -1,63 +0,0 @@
-###############################################################################
```
Этот файл не нужно удалять. Он тут и должен быть.

dha01 commented on .gitignore:
```diff
@@ -1,6 +0,0 @@
-packages
```
Этот файл не нужно удалять. Он тут и должен быть.

dha01 commented on SQL/Task_0_admin/test_query.sql:
```diff
@@ -0,0 +1,16 @@
+create table Dbo.test_test
```
Вот этого файла быть не должно.

dha01 commented on SQL/Task_61_DN/test_sql.txt:
```diff
@@ -0,0 +1 @@
+test
```
И этого файла тоже быть не должно.

dha01 commented on SQL/Task_14/create table Team.team.sql:
```diff
@@ -0,0 +1,23 @@
+		'2012-09-27'
+		)
+
+delete from Team.team
```
Удаление записи делать не нужно.

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Изменила название поля даты
Убрала удаление записи из таблицы

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Переименовала поле даты
Убрала ненужные файлы
Убрала удаление записи из таблицы
Переименовала везде speciality на specialty_detail

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Убрала лишние строки в запросе insert

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Окончательный вариант
Изменила название поля даты
Убрала запрос на удаление записи
Поменяла везде specialty на specialty_detail
Удалила ненужные файлы из папки SQL

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Вернула удаленные файлы

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание таблицы и схемы Team.team

Добавила комментарии к запросам в файле create table Team.team.sql
